index.html

```diff
@@ -956,17 +961,22 @@ <h3>They Trust Us</h3>
   <!-- <div id="preloader"></div> -->
 
   <!-- JavaScript Libraries -->
-  <script src="lib/jquery/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"
+    integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
   <script src="lib/jquery/jquery-migrate.min.js"></script>
   <script src="lib/bootstrap/js/bootstrap.bundle.min.js"></script>
   <script src="lib/easing/easing.min.js"></script>
   <script src="lib/mobile-nav/mobile-nav.js"></script>
-  <script src="lib/wow/wow.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/graingert-wow/1.2.2/wow.min.js"
+    integrity="sha256-gopJVYlh0S1SDzGgSVfVDKq3r27Lc8SzPE0FJ+c4EVk=" crossorigin="anonymous"></script>
   <script src="lib/waypoints/waypoints.min.js"></script>
   <script src="lib/counterup/counterup.min.js"></script>

```

Comment:
You missed this one :)

But don't worry it will be a good challenge to create our plugin, and then we will be good to get rid of both `waypoints` & `counterup`

Reply:
Thank you @Fcmam5 for helping me to complete my first pull request